glmnet/tests/test_linear.py

```diff
@@ -102,6 +102,15 @@ def test_alphas(self):
             m = m.fit(x, y)
             self.check_r2_score(y, m.predict(x), 0.90, alpha=alpha)
 
+    def test_coef_limits(self):
+            x, y = self.inputs[0]
+            lower_limits = 0
+            upper_limits = np.repeat(1,x.shape[1])
+            m = ElasticNet(lower_limits=lower_limits, upper_limits=upper_limits, random_state=5934)
+            m = m.fit(x, y)
+            assert(np.all(m.coef_) >= 0)
+            assert(np.all(m.coef_) <= 1)            
```

Comment:
As long as all coefficients are non-zero, these tests reduce to `assert True >= 0` and `assert True <= 1`. The closing parenthesis on the `np.all` needs to be after the integer.